core/web/resolver/p2p_test.go

```diff
@@ -0,0 +1,204 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/smartcontractkit/chainlink/core/services/keystore"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/p2pkey"
+)
+
+func TestResolver_GetP2PKeys(t *testing.T) {
+	t.Parallel()
+
+	query := `
+		query GetP2PKeys {
+			p2pKeys {
+				results {
+					id
+					peerId
+					publicKey
+				}
+			}
+		}
+	`
+
+	fakeKeys := []p2pkey.KeyV2{}
+	expectedKeys := []map[string]string{}
+	for i := 0; i < 2; i++ {
+		k, err := p2pkey.NewV2()
```

Comment:
you can use `p2pkey.MustNewV2XXXTestingOnly` in cases like this to generate cheaper keys from small seeds